os/net/lwip/src/core/tcp.c

```diff
@@ -540,6 +540,7 @@ struct tcp_pcb *tcp_listen_with_backlog(struct tcp_pcb *pcb, u8_t backlog)
 			if (lpcb->local_port == pcb->local_port) {
 				if (ip_addr_cmp(&lpcb->local_ip, &pcb->local_ip)) {
 					/* this address/port is already used */
+					errno = EADDRINUSE;
```

Comment:
using set_errno() and get_errno() is better than direct access to errno

Reply:
done